### ISSUE TITLE
Configure Lua error

### ISSUE THREAD

**Issue by CJ-Wright** [tschoonj/xraylib#24]
I get the following message when running the configure script.

```bash
checking for lua... /usr/bin/lua
checking lua.h usability... no
checking lua.h presence... no
checking for lua.h... no
checking lualib.h usability... no
checking lualib.h presence... no
checking for lualib.h... no
checking for exp in -lm... (cached) yes
checking for dlopen in -ldl... (cached) yes
checking for lua_load in -llua... no
configure: WARNING: Cannot build lua bindings
./configure: line 25716: syntax error near unexpected token `else'
./configure: line 25716: `		else'
```
Do you know what the issue is, and if it is on my end what I can do to fix it?

**Comment by tschoonj** [comment]
Hi Christopher,

Did you by any chance clone the repository and ran `autoreconf` on it to generate the configure script?
Which OS are you running?

**Comment by CJ-Wright** [comment]
Yes, I did.  I am using a cluster's copy of Scientific Linux release 6.5 (Carbon).  However, I was able to fix the issue with a wget.  Now I have the problem that anaconda doesn't know it exists since I didn't use pip or conda to install the package.  Anaconda doesn't really have a PYTHONPATH variable since it hardlinks everything.  What do you think I should do?

**Comment by tschoonj** [comment]
Scientific Linux must be too old to generate the correct configure script. I will increase the minimum required versions in configure.ac to avoid this in the future. 

Regarding your python issue: I am not much use here I am afraid as I don't know nearly enough python to help you with this... Why not install my [RPM xraylib-python packages](https://github.com/tschoonj/xraylib/wiki/Installation-instructions#fedora-centos-scientific-linux)? These should be picked up automatically by python and removes the need to compile xraylib manually completely. I don't think I have many any substantial changes since I released 3.1.0 anyway.

**Comment by CJ-Wright** [comment]
I got it to work with the source code, I need to use more --parameters than I expected.  Thank you very much for all you help.
